File: rating-service/src/test/java/com/software/modsen/ratingservice/component/RatingCucumberComponentTest.java
package com.software.modsen.ratingservice.component;

import io.cucumber.junit.Cucumber;
import io.cucumber.junit.CucumberOptions;
import org.junit.runner.RunWith;

@RunWith(Cucumber.class)
@CucumberOptions(
        features = "classpath:features",
        glue = "com/software/modsen/ratingservice/component"
)
public class RatingCucumberComponentTest {
}

File: passenger-service/src/test/kotlin/com/software/modsen/passengerservice/component/rating/RatingCucumberComponentTest.kt
package com.software.modsen.passengerservice.component.rating

import io.cucumber.junit.Cucumber
import io.cucumber.junit.CucumberOptions
import org.junit.runner.RunWith

@RunWith(Cucumber::class)
@CucumberOptions(
    features = ["classpath:features/passenger-rating"],
    glue = ["com/software/modsen/passengerservice/component/rating"]
)
class RatingCucumberComponentTest